doc_admin/AdminManual.adoc

```diff
@@ -428,15 +496,14 @@ To restore data with MySQL Workbench®:
 . Click the *Start Import* button at the right bottom corner of the window.
 
 
-==== 3.3.3 Backup of code and settings (Software)
+==== 2.3.3 Backup of OH code and settings
 
-Backup of the software *code and settings* are needed only if changes have been made to the “official version” to
-correctly perform any future Update process (see <<update-open-hospital,Update Open Hospital>>).
+Backup of the Open Hospital *code and settings* can be performed by saving the *"oh"* folder. Configuration files are placed under the *"oh/rsc"* folder.
 
-Backup of the client *data and configuration files*, especially for "PORTABLE" mode, can be performed by saving the *"data"* folder (including all subfolders) placed under the main OH installation folder.
+Backup of the client *data and configuration files*, especially for "PORTABLE" mode, can be performed by saving/copying the *"data"* folder (including all subfolders) placed under the main OH installation folder.
```

Comment:
I would change "placed under the main" to "found under the main"

Reply:
Should I replace all occurences of these sentence ?

Reply:
Did that also